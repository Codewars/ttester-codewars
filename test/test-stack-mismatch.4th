\ Copyright 2019 nomennescio
s" group of tests" describe#{
  s" single test" it#{
    <{ 0 0 -> 0 }>
  }#
  s" double test" it#{
    <{ 0 0 -> 0 }>
    <{ 0 -> 0 }>
  }#
  s" double fail test" it#{
    <{ 0 0 -> 0 }>
    <{ 0 -> 0 0 0 }>
  }#
}#
