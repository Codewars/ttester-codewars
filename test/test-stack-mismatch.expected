
<DESCRIBE::>group of tests

<IT::>single test

<FAILED::>Wrong number of results, expected 1 , got 2 

<COMPLETEDIN::>0.060 ms

<IT::>double test

<FAILED::>Wrong number of results, expected 1 , got 2 

<PASSED::>Test Passed

<COMPLETEDIN::>0.059 ms

<IT::>double fail test

<FAILED::>Wrong number of results, expected 1 , got 2 

<FAILED::>Wrong number of results, expected 3 , got 1 

<COMPLETEDIN::>0.071 ms

<COMPLETEDIN::>0.235 ms
